WikiWatershed/rapid-watershed-delineation
# Update API to support NHD

#### Overview

Exposes a new RWD API endpoint for NHD Region 2 data.
#### Testing Instructions
1. Checkout this branch
2. Checkout WikiWatershed/docker-rwd#19
3. From the `docker-rwd` project, run:
   
   > RWD_DATA=/opt/rwd-data/nhd ./scripts/run.sh
   
   (Where `RWD_DATA` points to a folder containing NHD Region 2 data; See #34)
4. Run:
   
   > curl http://localhost:5000/rwd-nhd/42.53903047967409/-77.55971981686719
#### Notes

Remaining work:
- Reproject output shapefiles from NAD83 to Latlng

Depends on WikiWatershed/docker-rwd#19

Connects #29
Connects #34
